### PR TITLE
Make @pazeshun happy by hand-eye calibration

### PR DESCRIPTION
**Pull request by wkentaro**
Close https://github.com/start-jsk/jsk_apc/issues/2204

## What is this?

We calibrated camera coordinates using interactive marker with @knorth55 .
This is fully supported by @YutoUchimi 's intrinsic camera parameter calibration, thanks. :bowing_man: 

**After**

**Before**

### CODE REVIEW

**Comment by wkentaro** [comment]
Close https://github.com/start-jsk/jsk_apc/issues/2204

File: jsk_arc2017_baxter/launch/setup/include/stereo_astra_hand.launch [hunk 1]
```diff
@@ -56,10 +54,24 @@
...
+        <node name="marker_6dof"
+              pkg="jsk_interactive_marker" type="marker_6dof">
+          <rosparam>
+            object_type: cube
+            frame_id: left_hand
+            publish_tf: true
+            tf_frame: left_hand_camera_left_depth_optical_frame
+            object_x: 0.1
+            object_y: 0.1
+            object_z: 0.1
```
Comment (pazeshun) [inline jsk_arc2017_baxter/launch/setup/include/stereo_astra_hand.launch]: Didn't you need to set initial_x , y, z and orientation here?

**Comment by pazeshun** [comment]
:joy

File: jsk_arc2017_baxter/launch/setup/include/stereo_astra_hand.launch (hunk 1 above)
Comment (wkentaro) [inline jsk_arc2017_baxter/launch/setup/include/stereo_astra_hand.launch]: actually, it seems `marker_6dof` has bug in the initialization of the pose from rosparam.
so I removed the initialization and it is set to 0 0 0 0 0 0.

File: jsk_arc2017_baxter/launch/setup/include/stereo_astra_hand.launch [hunk 2]
```diff
@@ -273,9 +285,31 @@
...
+            initial_x: 0.009
+            initial_y: -0.048
+            initial_z: 0.016
+            initial_orientation:
+              - -0.049
+              - -0.099
+              - 1.627
```
Comment (pazeshun) [inline jsk_arc2017_baxter/launch/setup/include/stereo_astra_hand.launch]: So you need to erase here, too?

**Comment by wkentaro** [comment]
```
executing command [sudo -H pip install -U -q gdown]
ERROR: the following rosdeps failed to install
  pip: Failed to detect successful installation of [gdown]
+EXIT_STATUS=1
+'[' 1 == 0 ']'
+sleep 30
+'[' 1 == 1 -a 2 -lt 3 ']'
+COUNT=3
+rosdep install -q -y --rosdistro indigo -n -q --ignore-src --skip-keys=jsk_smart_gui --skip-keys=ros3djs --skip-keys=pr2_calibration_launch --skip-keys=jsk_android_gui_api9 --skip-keys=ros2djs --skip-keys=face_recognition --skip-keys=roslibjs --skip-keys=force_proximity_ros --skip-keys=safe_teleop_base --from-paths /opt/ros/indigo/share .
executing command [sudo -H pip install -U -q gdown]
ERROR: the following rosdeps failed to install
  pip: Failed to detect successful installation of [gdown]
+EXIT_STATUS=1
+'[' 1 == 0 ']'
+sleep 30
+'[' 1 == 1 -a 3 -lt 3 ']'
+'[' 1 '!=' 0 ']'
+rosdep install -q -y --rosdistro indigo -n -q --ignore-src --skip-keys=jsk_smart_gui --skip-keys=ros3djs --skip-keys=pr2_calibration_launch --skip-keys=jsk_android_gui_api9 --skip-keys=ros2djs --skip-keys=face_recognition --skip-keys=roslibjs --skip-keys=force_proximity_ros --skip-keys=safe_teleop_base --from-paths /opt/ros/indigo/share .
executing command [sudo -H pip install -U -q gdown]
ERROR: the following rosdeps failed to install
  pip: Failed to detect successful installation of [gdown]
```

**Pull request merged**